eox_nelp/api_clients/tests/tests_certificates.py

```diff
@@ -0,0 +1,65 @@
+"""This file contains all the test for certificates api client file.
+
+Classes:
+    TestExternalCertificatesApiClient: Test for eox-nelp/api_clients/certificates.py.
+"""
+import unittest
+
+from django.utils import timezone
+from mock import patch
+
+from eox_nelp.api_clients.certificates import ExternalCertificatesApiClient
+from eox_nelp.api_clients.tests import BasicApiClientMixin
+
+
+class TestExternalCertificatesApiClient(BasicApiClientMixin, unittest.TestCase):
+    """Tests ExternalCertificatesApiClient"""
+
+    def setUp(self):
+        """Setup common conditions for every test case"""
+        self.api_class = ExternalCertificatesApiClient
+
+    @patch.object(ExternalCertificatesApiClient, "make_post")
+    @patch.object(ExternalCertificatesApiClient, "_authenticate")
+    def test_create_certificate(self, auth_mock, post_mock):
+        """Test successful post request.
+
+        Expected behavior:
+            - Response is the expected value
+        """
+        auth_mock.return_value = {}
+        expected_value = {
+            "status": {"success": True, "message": "successful", "code": 1}
```

Comment:
I assume this is an assumption of the response of auth endpoint? xD heheeh

Reply:
This really doesn't matter, this is the output of a mock  therefore that could be whatever you want, the important test here is that the output of `make_post` will be the output of the `create_external_certificate`